### DISCUSSION TITLE
Can't deploy a repository on Railway for no reason

### DISCUSSION THREAD

**Discussion by MikheilMusaelyan** [community/community#55238] [Other Feature Feedback, Questions, & Ideas]
So I am trying to deploy a repo on Railway.app, but I am getting an error which says: **Unfortunately, the attached GitHub account does not meet our good actor criteria.** But in GitHub settings, I can clearly see that Railway is installed, and my GitHub is very active too (I think they're blocking for less active accounts). I also have deployed my repo on Railway before. One problem that I can think of is that my Bearer token was visible for multiple commits, I was working on a project. But then I resolved everything, I don't know, that may be the case. 

Any advice?

**Comment by Tsoune71** [comment]
same problem

**Comment by abbasmusawis** [comment]
Same here

**Comment by electric-vehicle-optimisation** [comment]
same here

**Comment by Nicolascarames** [comment]
same here

**Comment by larsycoding** [comment]
same

**Comment by larsycoding** [comment]
use render

**Comment by mwt** [comment]
That sounds like overzealous anti-fraud measures

**Comment by walidkorchi** [comment]
any updates ?

**Comment by TusharKanjariya** [comment]
Same Problem with Github Account Verification

**Comment by Anubrata-Das** [comment]
Same here bro 🥲🥲

**Comment by sarcasacras** [comment]
yep, same here :D

**Comment by PhosRunia28** [comment]
same here 👯‍♂️

**Comment by tmozodbek** [comment]
Yeah, Same here

**Comment by Wnaz** [comment]
same here

**Comment by JamersonSouza** [comment]
same here

**Comment by f-paul** [comment]
Same !

**Comment by otabek-abdiraimov** [comment]
same :(

**Comment by VladOnMyOwn** [comment]
same

**Comment by rizkysulaeman** [comment]
Same :(

**Comment by krutoo** [comment]
same problem
